lib/rubocop/cop/rails/enum_syntax.rb

```diff
@@ -25,6 +25,7 @@ class EnumSyntax < Base
         MSG_OPTIONS = 'Enum defined with deprecated options in `%<enum>s` enum declaration. Remove the `_` prefix.'
         RESTRICT_ON_SEND = %i[enum].freeze
         OPTION_NAMES = %w[prefix suffix scopes default].freeze
```

Comment:
The following could be added at this point :-)

```suggestion
        OPTION_NAMES = %w[prefix suffix scopes default instance_methods].freeze
```

Reply:
Yeah. Looks like I somehow linked an older commit in the other PR.

I've added it. There is now also `validate` but that only works with the new syntax and shouldn't be considered here.